Makefile

```diff
@@ -153,7 +155,7 @@ build: generate fmt vet ## Build manager binary.
 
 .PHONY: run
 run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
+	go run ./main.go -metrics-bind-address ${METRICS_BIND_ADDRESS}
```

Comment:
Changes like this to the Makefile may make future rebases for operator-sdk more difficult. Customizing the Makefile is still fine of course I'm starting to try to keep all our customizations in a standard section. 

Wondering if looking for the ENV variable in main.go might end up being cleaner.